packages/dashboard/src/dashboard-actions/reverse-actions/reverseResize.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { reverseResize } from './reverseResize';
+import { MOCK_KPI_WIDGET } from '../../testing/mocks';
+
+it('inverts the changeInPosition attribute', () => {
+  expect(
+    reverseResize({
+      type: 'RESIZE',
+      payload: {
+        anchor: 'bottom',
+        changeInPosition: { x: 5, y: 3 },
+        widgetIds: [MOCK_KPI_WIDGET.id],
+        dashboardConfiguration: {
+          widgets: [MOCK_KPI_WIDGET],
+          viewport: { duration: '5m' },
+        },
+        cellSize: 10,
+        isActionComplete: true,
+      },
+    })
+  ).toEqual({
+    payload: {
+      anchor: 'bottom',
+      changeInPosition: { x: -5, y: -3 },
+      widgetIds: [MOCK_KPI_WIDGET.id],
+      dashboardConfiguration: {
+        widgets: [MOCK_KPI_WIDGET],
+        viewport: { duration: '5m' },
+      },
+      cellSize: 10,
+      isActionComplete: true,
+    },
+    type: 'RESIZE',
+  });
+});
+
+it('returns the original action when reversed twice', () => {
+  expect(
+    reverseResize(
+      reverseResize({
+        type: 'RESIZE',
```

Comment:
if we instead said `reverseResize(reversizeResize(action)).toEqual(action)` it would be a shorter, easier to read expression, rather than re-declaring twice